canonical-docs/snappy-docs
# Navigation to be fixed width and fixed mobile layout on language guides and Interface

Fixes #429 and #430 

## QA

1. Pull the branch or launch the demo
2. Make sure the navigation is fixed width across the site #429 
3. Check the [language guides](https://docs.snapcraft.io/build-snaps/languages) and [interface table](https://docs.snapcraft.io/reference/interfaces) can be seen on a mobile layout #430
